synapse/handlers/relations.py

```diff
@@ -153,7 +151,7 @@ async def get_relations(
 
     async def _get_bundled_aggregation_for_event(
         self, event: EventBase, user_id: str
-    ) -> Optional[BundledAggregations]:
+    ) -> Tuple[Optional[JsonDict], Optional[JsonDict]]:
```

Comment:
docstring needs an update

Reply:
does the method need renaming? it seems like it no longer returns the "bundled aggregation" [sic]?